### PR TITLE
MMT-951: On Metadata Draft Forms, clicking on the info button on a subform will open/close the accordion as well as showing the help modal.

### PR DESCRIPTION
**Pull request by ChristianTrummer99**
- Adjusted eui.js handler
- uncommented spec

### CODE REVIEW

File: vendor/assets/javascripts/eui-1.0.0/eui.js [hunk 1]
```diff
@@ -124,9 +124,11 @@
   // FUNCTIONALITY: #Accordion
 
     // Basic Accordion Functionality
-    $(".eui-accordion__header").click(function() {
-      $(this).siblings(".eui-accordion__body").slideToggle("fast");
-      $(this).closest(".eui-accordion").toggleClass("is-closed");
+    $(".eui-accordion__header").click(function(e) {
+      if (!$(e.target).is('i.eui-fa-info-circle')) {
```
Comment (charleshuang80) [inline vendor/assets/javascripts/eui-1.0.0/eui.js:128]: We should try to fix the issue without going into the vendor library (even if it is EUI). Did you look into doing that?

Comment (ChristianTrummer99) [reply vendor/assets/javascripts/eui-1.0.0/eui.js:128]: yes, this is the fix.

Comment (ChristianTrummer99) [reply vendor/assets/javascripts/eui-1.0.0/eui.js:128]: I'll take another pass at it, and let you know if I can come up with anything else

File: app/assets/javascripts/accordion.coffee [hunk 2]
```diff
@@ -2,6 +2,18 @@ $(document).ready ->
   # Disable toggle
   $('.eui-accordion__header.disable-toggle').unbind('click')
 
+  # the eui-accordion__header listener in eui.js that is responsible for opening/closing the accordions
+  # allows the accordions to open/close when the help icon is clicked. Performing this unbind/rebind
+  # operation allows the below click listener to take priority and achieves the desired accordion/help
+  # icon behavior without editing the vendor file (eui.js)
+  $('.eui-accordion__header').not('.eui-accordion__header.disable-toggle').unbind('click')
+  $('.eui-accordion__header').not('.eui-accordion__header.disable-toggle').bind('click')
```
Comment (charleshuang80) [inline app/assets/javascripts/accordion.coffee:10]: Is this line necessary? Doesn't what you do on line 12 do the binding?

**Review by william-valencia — APPROVED** [review]

**Review by jdf-eos — APPROVED** [review]

**Pull request merged**